front/app/modules/commercial/insights/admin/containers/Insights/Edit/InputsTable/InputsTable.test.tsx

```diff
@@ -205,4 +204,24 @@ describe('Insights Input Table', () => {
       screen.getByText('You have no input assigned to this category yet')
     ).toBeInTheDocument();
   });
+  it('filters table by category', () => {
+    mockLocationData = { pathname: '', query: { category: 'category' } };
+
+    render(<InputsTable />);
+    expect(useInsightsInputs).toHaveBeenCalledWith(viewId, {
+      category: 'category',
+      search: undefined,
+      pageNumber: NaN,
```

Comment:
Do we really want pageNumber: NaN ? Might be good to add a little check and simply not have it ? (I know, I introduced the issue myself, sorry.)

Reply:
Good point!